app/controllers/bands_controller.rb

```diff
@@ -1,10 +1,46 @@
 class BandsController < InheritedResources::Base
 
+  before_action :find_band, except: [:index, :new, :create]
```

Comment:
Yeah, you're right. Let's keep it like this. For some reason I just think that this callback was working only for the `new & edit` actions. This is great in this way.